### PR TITLE
http: scatter table API for table handler

### PR DESCRIPTION
**Pull request by nolouch**
should after https://github.com/pingcap/pd/pull/1037

### CODE REVIEW

**Comment by jackysp** [comment]
Need a test case for it.

File: server/http_handler.go [hunk 1]
```diff
@@ -647,6 +653,115 @@ func (h ddlHistoryJobHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
...
+	}
+	return nil
+}
+
+func (h tableHandler) deleteScatterSchedule(name string) error {
+	pdAddrs, err := h.getPDAddr()
+	if err != nil {
+		return err
+	}
+	scheduleURL := fmt.Sprintf("http://%s/pd/api/v1/schedulers/scatter-range-leader-%s", pdAddrs[0], name)
```
Comment (tiancaiamao) [inline server/http_handler.go]: IMHO, this API design is bad
Prefer Post to Get, and you can use simpler http client API

File: server/http_status.go [hunk 2]
```diff
@@ -58,6 +58,8 @@ func (s *Server) startHTTPServer() {
 	if s.cfg.Store == "tikv" {
 		// HTTP path for tikv
 		router.Handle("/tables/{db}/{table}/regions", tableHandler{tikvHandlerTool, opTableRegions})
+		router.Handle("/tables/{db}/{table}/scatter", tableHandler{tikvHandlerTool, opTableScatter})
```
Comment (ciscoxll) [inline server/http_status.go]: need to add the test in `http_handler_test.go`.

**Comment by tiancaiamao** [comment]
LGTM

**Comment by tiancaiamao** [comment]
DNM: this one should after pingcap/pd#1037

**Pull request merged**